include/debug/object_tracing_common.h

```diff
@@ -20,6 +20,20 @@
 #define SYS_TRACING_OBJ_INIT_DLL(name, obj) do { } while (false)
 #define SYS_TRACING_OBJ_REMOVE_DLL(name, obj) do { } while (false)
 
+/**
+ * @note	Extended tracing macro blanks
+ */
+#define SYS_PORT_TRACING_FUNC(type, func, ...) do { } while (false)
```

Comment:
Absolutely, it would be possible to put them in tracing.h but it is fairly stuffed at the moment. We could go for a separate header file in the same directory as tracing.h if that is fine with you guys.

Reply:
yes, this should be fine.

Reply:
These macros have now been migrated into a tracing_macros.h file in the tracing directory.